packages/engine/src/gltf/GLTFLoaderFunctions.ts

```diff
@@ -783,6 +786,20 @@ const loadMaterial = async (options: GLTFParserOptions, materialIndex: number) =
 
   await Promise.all(promises)
 
+  //apply deltas
+  const deltaState = getState(SceneDeltaState)
+  const sourceDelta = deltaState[options.documentID.replaceAll(/\?hash=[^-]+/g, '')]
+  if (sourceDelta) {
+    const nodeID = getComponent(materialEntity, NodeIDComponent)
+    const nodeDelta = sourceDelta[nodeID]
+    if (nodeDelta) {
+      const materialDelta = nodeDelta['materialParameters']
```

Comment:
let's define either a const or a global type override for `'materialParameters'` rather than hardcoding a magic string